plugins/orbslam2_plugin/orbslam2_sensors.py

```diff
@@ -0,0 +1,105 @@
+import os
+import math
+import yaml
+from typing import Any, Optional
+from tempfile import mkstemp
+
+import gym
+import numpy as np
+
+import orbslam2
+from core.base_abstractions.task import SubTaskType, Task
+from core.base_abstractions.sensor import Sensor, RGBSensor, DepthSensor
+from core.base_abstractions.misc import EnvType
+from plugins.robothor_plugin.robothor_environment import RoboThorEnvironment
+from plugins.robothor_plugin.robothor_sensors import RGBSensorRoboThor, DepthSensorRoboThor, GPSCompassSensorRoboThor
+from plugins.robothor_plugin.robothor_tasks import PointNavTask
+from plugins.orbslam2_plugin.orbslam2_util import state_to_pose, matrix_to_euler_angles
+from utils.misc_utils import prepare_locals_for_super
+
+
+# check Camera.fps, Camera.bf, ThDepth, DepthMapFactor
+# parameterize ORBextractor
+def init_settings(w, h, fov):
```

Comment:
Would be great to have type hints for these values. Also please run the `scripts/auto_format.sh` script.